From 95a161ed424181bfca8da6ae6d14a38cf1b04458 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Thu, 12 Jan 2023 16:00:53 +0100
Subject: [PATCH 0339/1695] drivers: stm32_exti: stm32mp_exti: align extern
 prototype

The name of the parameter in the prototype exported by the include
file does not match the comment above nor the exported function in
the driver code.

Fix the name of the parameter.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git bda81a1eb4e5c16aff2d4555775d75e35dd6c88b drivers: stm32_exti: stm32mp_exti: align extern prototype]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: Id81f9213619fecf8b6c797538a5ff0b9e51f5961
---
 core/include/drivers/stm32_exti.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/include/drivers/stm32_exti.h b/core/include/drivers/stm32_exti.h
index 7a758d16c..afb246c0f 100644
--- a/core/include/drivers/stm32_exti.h
+++ b/core/include/drivers/stm32_exti.h
@@ -18,7 +18,7 @@
  * @type: type (rising, falling or both) to set
  * Return a TEE_Result compliant return value
  */
-void stm32_exti_set_type(uint32_t exti, uint32_t type);
+void stm32_exti_set_type(uint32_t exti_line, uint32_t type);
 
 /*
  * Mask EXTI Interrupt (IMR)
-- 
2.47.2

